bolt-contracts/src/contracts/BoltChallenger.sol

```diff
@@ -31,7 +31,7 @@ contract BoltChallenger is IBoltChallenger {
     // ========= CONSTANTS =========
 
     /// @notice The challenge bond required to open a challenge.
-    uint256 public constant CHALLENGE_BOND = 1 ether;
+    uint256 public constant COMMITMENT_BOND = 1 ether;
```

Comment:
Why was this renamed? I think `CHALLENGE_BOND` is more clear since it's the bond required to open a challenge